client/controller/build.gradle

```diff
@@ -50,6 +50,9 @@ dependencies {
 
     // https://mvnrepository.com/artifact/org.postgresql/postgresql
     compile group: 'org.postgresql', name: 'postgresql', version: '9.4-1201-jdbc41'
+
+    //local jar packege for oracle
```

Comment:
add space between // and text, and 'package' is misspelled. Also, delete space after { on next line.